src/gui/editors/SongEditor.cpp

```diff
@@ -547,19 +558,19 @@ void SongEditor::wheelEvent( QWheelEvent * we )
 		{
 			z--;
 		}
-		z = qBound( 0, z, m_zoomingModel->size() - 1 );
+		z = qBound(0, z, m_zoomingXModel->size() - 1);
```

Comment:
```suggestion
		z = std::clamp(z, 0, m_zoomingXModel->size() - 1);
```
We're trying to get away from Qt-specific functions where we can. You may need to include <algorithm> for this.